cheddar/cheddar-context/src/main/java/com/clicktravel/cheddar/request/context/DefaultSecurityContext.java

```diff
@@ -26,30 +26,33 @@
     private final Optional<String> userId;
     private final Optional<String> teamId;
     private final Optional<String> identityProviderId;
+    private final Optional<String> agentIdentityProviderId;
     private final Optional<String> agentUserId;
 
     private final Optional<String> appId;
 
     public DefaultSecurityContext(final String userId, final String teamId,
-            final String agentUserId, final String appId, final String identityProviderId) {
+            final String agentUserId, final String appId, final String identityProviderId,
```

Comment:
This is removing a DefaultSecurityContext with the signature that has 

```
DefaultSecurityContext(final String userId, final String teamId,
            final String agentUserId, final String appId, final String identityProviderId)
```

Which I think is a break to the interface, but I think since that was added in the last version of Cheddar any release has been made so it shouldn't cause any issue, and as this feature is not/(only just in prod) I highly doubt anyone has used it 🤔   So it's almost certainly 100% safe to break the interface :) 

I only know this because I did this exact same thing last week and found out when I ran UsersJava Service Tests 😂